### DISCUSSION TITLE
Vercel's restriction on phone number usage

### DISCUSSION THREAD

**Discussion by ohad-ironblocks** [vercel/community#2635] [Idea]
I have 2 Github accounts, a personal one and one linked to my work's email.
Because I already opened a Vercel account on my personal email and because I don't have 2 sims... I can't open another account.

Because of this restriction, I can't use Vercel to host any of our apps.
Please remove the phone number restriction.

**Comment by linglong-blockus** [comment]
Is anyone helping with this? this is blocking our development as well!

**Reply by xav1ne** [reply]
would love to have this aliased
